.github/workflows/license_check.yml

```diff
@@ -0,0 +1,20 @@
+name: Check for LICENSE File in femr Directory
+
+on:
+  push:
+    branches:
+      - temr-dev-noah
```

Comment:
I think maybe you can just remove the "branches:" portion, and it will do it for any push regardless of branch? Not 100%